entrypoint.sh.addHostUserToContainer

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# Add local user
+# Either use the LOCAL_USER_ID if passed in at runtime or
+# fallback
+
+USER_ID=${LOCAL_USER_ID:-9001}
```

Comment:
the script does run after container construction, but before the CMD executes as per the specification of ENTRYPOINT